challenges/challenge-cowsay-two/solution2.js

```diff
@@ -5,14 +5,46 @@
 // =================
 
 // 1. Make  a command line interface.
+//To accept arguments from command line we can use readline module and get input from process.stdin
+//The question method has two parameters. The first one is the question and the second one is the user input.
+//It calls the callback function once the user enters input.
+const readline = require('readline');
+const rl = readline.createInterface({
```

Comment:
I like your code overall. One minor question: what 'rl' means? I guess it is 'readline' but you already had this variable name and used 'rl'. Please, try to avoid abbreviations in your variable names. Please, think of a better name here. Other than that your code looks good.

Reply:
Thank you for the feedback. I see your point. I'll change rl to readlineInterface to make the variable name clearer.